packages/system-tests/assets/sfdx-simple/functions/demoJavaScriptFunction/package.json

```diff
@@ -23,7 +23,7 @@
     "@types/chai-as-promised": "^7.1.0",
     "@types/mocha": "^5.2.5",
     "@types/node": "^10.12.18",
-    "@types/sinon": "^7.0.4",
+    "@types/sinon": "^7.5.2",
```

Comment:
I don't offhand.  I know there is an option in the vscode extensions repo to run with the sfdx-simple package.  This may be the template for that flow.

Reply:
oh! I actually know more now :).  When working with @RitamAgrawal on updating the LWC tests she was doing some integration tests.  There are basically used as part of the integration tests when we need a sfdx project to test against.  I'm not sure if there are function specific integration tests but I bet that's why these exist. I suspect this particular change probably doesn't matter for that use case.